src/main/java/ch/njol/skript/events/EvtHarvest.java

```diff
@@ -0,0 +1,68 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.events;
+
+import org.bukkit.block.Block;
+import org.bukkit.event.Event;
+import org.bukkit.event.player.PlayerHarvestBlockEvent;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptEvent;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+
+public class EvtHarvest extends SkriptEvent {
+
+	static {
+		if (Skript.classExists("org.bukkit.event.player.PlayerHarvestBlockEvent"))
+			Skript.registerEvent("Harvest", EvtHarvest.class, PlayerHarvestBlockEvent.class, "[player] [block|crop] harvest[ing] [[of] %-itemtypes%]")
+					.description("This event is called whenever a player harvests a block.",
+							"A 'harvest' is when a block drops an item (usually some sort of crop) and changes state, but is not broken in order to drop the item.")
+					.examples("on player harvesting:",
+							"\tmessage \"You harvested %block% which drops %event-items% from your %event-slot%!\"")
```

Comment:
It doesn't drop "from" the player's hand, the player harvests using an item, or with an item.
```suggestion
							"\tmessage \"You harvested %block% which drops %event-items%, using your %event-slot%!\"")
```